### ISSUE TITLE
Issue with Unidiffuser pipeline implementation in diffusers repository

### ISSUE THREAD

**Issue by SKDDJ** [dg845/diffusion-models#1]
Hi there,

I'm having some trouble with the Unidiffuser pipeline implementation in the diffusers repository. Specifically, I'm trying to use the sample code provided in the README to generate images using the Unidiffuser pipeline, but all of the images that I'm generating are coming out black, regardless of whether or not I input a prompt.

I was wondering if you could provide me with some guidance on how to properly use the Unidiffuser pipeline, or if there might be a bug in the implementation that's causing this issue.

Thanks in advance for your help!

**Comment by dg845** [comment]
Hi @SKDDJ, I think you might have opened an issue in the wrong repository; [huggingface/diffusers](https://github.com/huggingface/diffusers) might be a better place for this issue.

For the [UniDiffuser pipeline](https://huggingface.co/docs/diffusers/api/pipelines/unidiffuser), there is currently an issue with numerical stability on PyTorch 1.X, where the image latents become `NaN`s by the end of the denoising loop. This issue can be mitigated by switching to PyTorch 2.X.

If you're experiencing this issue on PyTorch 2.X, could you open a new issue with details at [huggingface/diffusers](https://github.com/huggingface/diffusers)? I will try to respond promptly there :).

**Comment by SKDDJ** [comment]
@dg845 ,Thank you for your reply. I apologize for the confusion about opening the issue in the wrong repository. I actually intended to open an issue in your "diffusers" repository forked by "huggingface/diffusers". However, I realized that I cannot do so.

Regarding the issue with numerical stability on PyTorch 1.X, I appreciate your suggestion to switch to PyTorch 2.X. I will try that and let you know if it resolves the problem. If I encounter any issues on PyTorch 2.X, I will open a new issue with details at huggingface/diffusers, as suggested.

Thanks again for your help and support :)
